package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "yaml-ast-parser-custom-tags",
+  "name": "yaml-language-server-parser",
   "version": "0.0.43",
```

Comment:
@evidolob @gorkem Do you think we should reset the version number to 0.0.1 or should we just keep it as is?

Reply:
I do not see an issue with resetting the version

Reply:
Same for me.